roles/kubernetes/master/defaults/main/main.yml

```diff
@@ -83,25 +79,6 @@ audit_webhook_mode: batch
 audit_webhook_batch_max_size: 100
 audit_webhook_batch_max_wait: 1s
 
-# Limits for kube components
-kube_controller_memory_limit: 512M
-kube_controller_cpu_limit: 250m
-kube_controller_memory_requests: 100M
-kube_controller_cpu_requests: 100m
-kube_controller_node_monitor_grace_period: 40s
-kube_controller_node_monitor_period: 5s
-kube_controller_pod_eviction_timeout: 5m0s
-kube_controller_terminated_pod_gc_threshold: 12500
-kube_scheduler_memory_limit: 512M
-kube_scheduler_cpu_limit: 250m
-kube_scheduler_memory_requests: 170M
-kube_scheduler_cpu_requests: 80m
-kube_apiserver_memory_limit: 2000M
-kube_apiserver_cpu_limit: 800m
-kube_apiserver_memory_requests: 256M
-kube_apiserver_cpu_requests: 100m
-kube_apiserver_request_timeout: "1m0s"
```

Comment:
As you can see in the failing tests, this one is still used

Reply:
Yeah thanks, it was lurking next to the others and I was a bit quick...